### PR TITLE
update plugin development

### PR DESCRIPTION
**Pull request by 6293**

### CODE REVIEW

File: en_US/advanced/plugins.md [hunk 1]
```diff
@@ -13,31 +13,30 @@ You can compile it with the [emqx](https:://github.com/emqx/emqx) core project a
...
+| [emqx_bridge_mqtt](https://github.com/emqx/emqx/tree/master/apps/emqx_bridge_mqtt) | etc/plugins/emqx_bridge_mqtt.conf     | MQTT Message Bridge Plugin         |
+| [emqx_coap](https://github.com/emqx/emqx/tree/master/apps/emqx_coap)               | etc/plugins/emqx_coap.conf            | CoAP protocol support              |
+| [emqx_lwm2m](https://github.com/emqx/emqx/tree/master/apps/emqx_lwm2m)             | etc/plugins/emqx_lwm2m.conf           | LwM2M protocol support             |
+| [emqx_sn](https://github.com/emqx/emqx/tree/master/apps/emqx_sn)                   | etc/plugins/emqx_sn.conf              | MQTT-SN protocol support           |
+| [emqx_stomp](https://github.com/emqx/emqx/tree/master/apps/emqx_stomp)             | etc/plugins/emqx_stomp.conf           | Stomp protocol support             |
+| [emqx_recon](https://github.com/emqx/emqx/tree/master/apps/emqx_recon)             | etc/plugins/emqx_recon.conf           | Recon performance debugging        |
+| [emqx_reloader](https://github.com/emqx/emqx/tree/master/apps/emqx_reloader)       | etc/plugins/emqx_reloader.conf        | Hot load plugin                    |
```
Comment (zmstone) [inline en_US/advanced/plugins.md]: this is deleted from 4.3

File: en_US/advanced/plugins.md [hunk 2]
```diff
@@ -13,31 +13,30 @@ You can compile it with the [emqx](https:://github.com/emqx/emqx) core project a
...
-| [emqx_stomp](https://github.com/emqx/emqx-stomp)             | etc/plugins/emqx_stomp.conf           | Stomp protocol support             |
-| [emqx_recon](https://github.com/emqx/emqx-recon)             | etc/plugins/emqx_recon.conf           | Recon performance debugging        |
-| [emqx_reloader](https://github.com/emqx/emqx-reloader)       | etc/plugins/emqx_reloader.conf        | Hot load plugin                    |
-| [emqx_plugin_template](https://github.com/emqx/emqx-plugin-template) | etc/plugins/emqx_plugin_template.conf | plugin develop template            |
+| [emqx_dashboard](https://github.com/emqx/emqx/tree/master/lib-ce/emqx_dashboard)   | etc/plugins/emqx_dashbord.conf        | Web dashboard Plugin (Default)     |
+| [emqx_management](https://github.com/emqx/emqx/tree/master/apps/emqx-management)   | etc/plugins/emqx_management.conf      | HTTP API and CLI Management Plugin |
+| [emqx_auth_clientid](https://github.com/emqx/emqx/tree/master/apps/emqx-auth-clientid) | etc/plugins/emqx_auth_clientid.conf   | ClientId Auth Plugin               |
```
Comment (zmstone) [inline en_US/advanced/plugins.md]: deleted from 4.3
```suggestion

```

File: en_US/advanced/plugins.md [hunk 3]
```diff
@@ -13,31 +13,30 @@ You can compile it with the [emqx](https:://github.com/emqx/emqx) core project a
...
+| [emqx_auth_username](https://github.com/emqx/emqx/tree/master/apps/emqx-auth-username) | etc/plugins/emqx_auth_username.conf   | Username/Password Auth Plugin      |
```
Comment (zmstone) [inline en_US/advanced/plugins.md]: deleted from 4.3
```suggestion

```

File: en_US/advanced/plugins.md [hunk 4]
```diff
@@ -231,34 +230,42 @@ plugin_name.key = value
...
+```
+
+#### Run your code
+
+Start the node (interactive mode)
+```
+./_build/emqx/rel/emqx/bin/emqx console
+```
+
+Load the plugin with command:
```
Comment (k32) [inline en_US/advanced/plugins.md]: Nit: the command

File: en_US/advanced/plugins.md [hunk 5]
```diff
@@ -231,34 +230,42 @@ plugin_name.key = value
...
+```
+./_build/emqx/rel/emqx/bin/emqx_ctl plugins load plugin_name
+```
+
+::: tip Tip
+To have the plugin enabled/loaded by default, you can include it by `data/loaded_plugins.tmpl`.
```
Comment (k32) [inline en_US/advanced/plugins.md]: include it by -> include it in?

**Review by zmstone — APPROVED** [review]

**Pull request merged**